understanding/22/target-size-minimum.html

```diff
@@ -52,27 +50,31 @@ <h2>Intent of Target Size (Minimum)</h2>
-            <li>If the spacing of the targets is essential to the information being conveyed, the "Essential" exception applies. For example, in digital maps, the position of pins is analogous to the position of places shown on the map. If there are many pins close together, the offset between pins and neighboring pins will often be below 24 CSS pixels. It is essential to show the pins at the correct map location, therefore the Essential exception applies. As well, some jurisdictions legally require online forms to replicate paper forms, which can impose constraints on the size of targets. In such jurisdictions, any legal requirement to replicate small targets can be considered essential.</li>
-            <li>Browsers have default renderings of some controls, such as the days of the month calendar in an <code>input type="date"</code>. As long as the author has not modified the user agent default, the target size for a "User agent control" is excepted.</li>
-            <li>In cases where a target does not have an area equivalent to 24 by 24 CSS pixels, but there is another control that can achieve the underlying function that <em>does</em> meet the minimum area, the target can be excepted based on the "Equivalent" exception.</li>
+            <li><strong>Spacing:</strong> If a target is smaller than 24 by 24 CSS pixels but has sufficient spacing, it passes. The spacing is sufficient if the offset between targets is at least 24 CSS pixels. <a href="#dfn-target-offset">Target offset</a> measures the distance between targets, measured from the farthest side of one target to the nearest side of the adjacent target. It assessed for each target, to each nearby target. What edge of the target is <em>farthest</em> depends on the relative position of the neighboring target. For example, if considering distance above the target, the farthest edge would be the bottom of the target, and the offset would be measured from this bottom edge to the bottom edge of the neighboring target above. Note that where the targets differ in size or shape, the offset between targets may also differ. The offset distance must be at least 24 CSS pixels in all cases to fall under this exception.</li>
```

Comment:
We should be consistent about using the full term "target offset" instead of "offset" or "offset distance" - it has a very specific meaning which is different from what most people would think of as an "offset". I think it might also be a good idea to avoid using "farthest" and "nearest" language here, based on the confusion we saw in #2695. Maybe something like this?

```suggestion
            <li><strong>Spacing:</strong> If a target is smaller than 24 by 24 CSS pixels but has sufficient spacing, it passes. The spacing is sufficient if the <a href="#dfn-target-offset">target offset</a> between targets is at least 24 CSS pixels. Target offset is the length of the longest possible line that starts at an edge of a target (A), intersects a second edge of A, and ends at the closest edge of a second target (B). It is assessed <em>separately</em> from a target to each other nearby target; the target offset must be at least 24 CSS pixels to <em>all</em> nearby targets to fall under this exception. The line used to measure target offset depends on the sizes, shapes, and relative positions of the two targets. For example, if considering the target offset from a rectangular target (A) to an equally-sized rectangular neighbor target (B) directly to A's right, target offset would be measured with a horizontal line starting from the left edge of A, passing through the right edge of A, and ending at the left edge of B. Note that the target offset from A to B may be different than the target offset from B to A.</li>
```

Reply:
(maybe also s/offset/target offset in the caption that begins with "Fail: two rows of buttons")

Reply:
That looks good, apart from the "separately" bit. I don't think it is clear what is separate in that sentence.

I think this would mean what you intended?
`It is assessed from a target to <em>each</em> other nearby target;`